common/transforms/und-Ethi-t-und-latn-m0-ies-jes-1964-geminate.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE supplementalData SYSTEM "../../common/dtd/ldmlSupplemental.dtd">
+<supplementalData>
+	<version number="$Revision$"/>
+	<transforms>
+		<transform source="Ethi" target="Latn" variant="IES_JES_1964_Geminate" direction="forward" draft="contributed" alias="Ethiopic-Latin/IES_JES_1964-Geminated und-Latn-t-und-ethi-m0-iesjes-1964-geminate">
+			<tRule><![CDATA[
+########################################################################
+#
+# Preemptive gemination for the Encylopedia Aethiopica Transliteration system.
+#
+# This transformation applies the Ethiopic-to-Ethiopic gemination mappings prior to
+# performing the regular Ethiopic-to-IES/JES 1964-Latin transform.  These mappings
+# are supported in the forward direction (Ethiopic-to-Latin) only at this time.
+#
+# Documentation for these transforms are given in the Ethiopic-Gemination.xml,
+# and Ethiopic-Latin-Aethiopica.xml files respectively.
```

Comment:
where are those two files?

Reply:
Thanks for pointing that out, the references to file names didn't get cleaned up in the comments after renaming occurred to a BCP47 style convention.  I'll fix this in the comments.  In this particular case, the dependent file names are now:

`am-Ethi-t-am-ethi-m0-geminate.xml`

and

`und-Ethi-t-und-latn-m0-ies-jes-1964-geminate.xml`

respectively.

Reply:
My last commit corrected the filenames in comments.